vision.py

```diff
@@ -224,17 +215,17 @@ def run(self):
                 self.CameraManager.sinks[0].getError()
             )
         else:
-
+            self.frame = cv2.flip(self.frame, -1)
```

Comment:
Is -1 the magic number for flipping the way you want it flipped?

Reply:
Yes.